### DISCUSSION TITLE
Email notification

### DISCUSSION THREAD

**Discussion by Dissorder** [MISP/MISP#8033] [Q&A]
### I cannot activate email notifications

Hi, I has been triying to enable email notifications, but no email is send. Also a tried to send a message from 'Administration > Contact Users' and show this error: 

![imagen_2021-12-15_220642](https://user-images.githubusercontent.com/47428603/146306398-5d2cc02a-f153-43f8-b922-bd663ff6f34f.png)
> E-mails sent, but failed to deliver the messages to the following recipients: ...

I changed the value of  `MISP.disable_emailing` to `false` and still failing. Can anybody help me to understand whats going on? If I'm doing something wrong or need to make other changes in the configuration. Thanks.

OS: `Ubuntu Server 20.04.3`
MISP version: `2.4.150`

**Comment by righel** [comment]
Hello @Dissorder ,
Could you please create an issue instead of a discussion?
